workers_kv.go

```diff
@@ -83,18 +84,53 @@ func (api *API) CreateWorkersKVNamespace(ctx context.Context, req *WorkersKVName
 //
 // API reference: https://api.cloudflare.com/#workers-kv-namespace-list-namespaces
 func (api *API) ListWorkersKVNamespaces(ctx context.Context) (ListWorkersKVNamespacesResponse, error) {
-	uri := fmt.Sprintf("/accounts/%s/storage/kv/namespaces", api.AccountID)
+	uri := fmt.Sprintf("/accounts/%s/storage/kv/namespaces?per_page=100", api.AccountID)
 	res, err := api.makeRequestContext(ctx, http.MethodGet, uri, nil)
 	if err != nil {
 		return ListWorkersKVNamespacesResponse{}, errors.Wrap(err, errMakeRequestError)
 	}
 
-	result := ListWorkersKVNamespacesResponse{}
-	if err := json.Unmarshal(res, &result); err != nil {
-		return result, errors.Wrap(err, errUnmarshalError)
+	overallResult := ListWorkersKVNamespacesResponse{}
+	if err := json.Unmarshal(res, &overallResult); err != nil {
+		return overallResult, errors.Wrap(err, errUnmarshalError)
 	}
 
-	return result, err
+	totalPageCount := overallResult.ResultInfo.TotalPages
+	var wg sync.WaitGroup
+	wg.Add(totalPageCount - 1)
+	errc := make(chan error)
+
+	for i := 2; i <= totalPageCount; i++ {
```

Comment:
In #288, we seem to duplicate the request for the first page of zones. Maybe there's a reason for that? Here, we don't duplicate the request, so I'm calling that out.

Reply:
If I recall, it was for requests that only had a single page which wasn't getting returned otherwise. Memory is fuzzy though. An approach like #389 might be good to follow here. We can get #288 updated in another PR.